core/bones/date.py

```diff
@@ -33,21 +32,26 @@ def __init__(
         """
         super().__init__(**kwargs)
 
-        if creationMagic or updateMagic:
-            self.readonly = True
-
-        self.creationMagic = creationMagic
-        self.updateMagic = updateMagic
-
+        # Either date or time must be set
         if not (date or time):
-            raise ValueError("Attempt to create an empty datebone! Set date or time to True!")
+            raise ValueError("Attempt to create an empty DateBone! Set date or time to True!")
 
+        # Localize-flag only possible with date and time
         if localize and not (date and time):
             raise ValueError("Localization is only possible with date and time!")
+        # Default localize all DateBones, if not explicitly defined
+        elif localize is None and date and time:
+            localize = True
 
-        if self.multiple and (creationMagic or updateMagic):
-            raise ValueError("Cannot be multiple and have a creation/update-magic set!")
+        # Magic is only possible in non-multiple bones and why ever only on readonly bones...
+        if creationMagic or updateMagic:
+            if self.multiple:
+                raise ValueError("Cannot be multiple and have a creation/update-magic set!")
+
+            self.readonly = True  # todo: why???
```

Comment:
I think this a quick & dirty implementation.
A good implementation would mark this field as `readonly` on `add` if `creationMagic` is enabled and on `edit` if `updateMagic` is enabled.

Reply:
Well, IMHO we generally could get rid of anything "magic" in VIUR.  The behavior is as before, I don't want to change it within this PR. We generally should think about a customizable `on_add`- and `on_edit`-trigger for bones which calls a function providing a value.